tests/e2e/kubernetes/kubernetes_test.go

```diff
@@ -152,9 +152,8 @@ func TestKubernetesHAModeMTLSEnabled(t *testing.T) {
 	})...)
 
 	tests = append(tests, common.GetTestsOnUninstall(currentVersionDetails, common.TestOptions{
-		UninstallAll: true,
```

Comment:
Basically, `ComponentsTestOnInstallUpgrade` is failing if we have more tests after `UninstallAll`. This was not the case earlier.
Has there been any change in uninstall --all related to removal of resources or components?

Reply:
I have also modified some certificate renewal tests, there also I was noticing time out because of a flawed logic in my tests and I was having 3 time taking tests in one test block.

Reply:
corrected all tests